ext.php

```diff
@@ -20,4 +20,19 @@
 */
 class ext extends \phpbb\extension\base
 {
+	/**
+	 * Check whether or not the extension can be enabled.
+	 * The current phpBB version should meet or exceed
+	 * the minimum version required by this extension:
+	 *
+	 * Require phpBB 3.1.2 due to the updated INCLUDECSS syntax.
+	 *
+	 * @return bool
+	 * @access public
+	 */
+	public function is_enableable()
+	{
+		$config = $this->container->get('config');
+		return version_compare($config['version'], '3.1.2', '>=');
```

Comment:
Seems good for me. This check will prevent many help questions, I would guess.
But do you want to use hardcoded version number here? Maybe it's easier, but I tend to forget that and just change composer.json :D We could read the composer metadata here, but We would have to split the phpbb version string. Hmh.
Maybe it's easier to just hardcode it, yeah.